components/tidb_query_expr/src/impl_cast.rs

```diff
@@ -1352,6 +1345,37 @@ fn cast_json_as_bytes(ctx: &mut EvalContext, val: Option<JsonRef>) -> Result<Opt
     }
 }
 
+#[rpn_fn(nullable, capture = [ctx])]
```

Comment:
Myabe we can remove `nullable` here? `rpn_fn` should already implement it.

Reply:
Seems it cannot compile if I remove `nullable`.

Reply:
OK, let's figure it out in another PR.

Reply:
I didn't implement non-nullable with context support